### DISCUSSION TITLE
Woche 7

### DISCUSSION THREAD

**Discussion by TimSchnur** [Christian-2003/CrInGE#29] [Student Exchange]
Hallo zusammen,

in dieser Woche stand die Erweiterung der Kernfunktionen an.
Deshalb haben wir die Entwicklung der Karten als absolute Grundlage für das 🎮 Spiel in Angriff genommen.
Dies beinhaltet sowohl das 💾 Speichern und Lesen diverser Datenstrukturen, als auch das Bearbeiten der Karte im 📝 Editor und Anzeigen der Karte mittels Game 👁️ Renderer.

Allerdings ist dies keine kleine Aufgabe, weshalb wir in dieser Woche keine neue demonstrierbare Demo haben. Die Funktionen sind nach wie vor in Arbeit, wobei das 💾 Persistieren bereits größtenteils fertiggestellt ist. Deshalb werden wir dieses Feature auch nächste Woche noch weiter implementieren.

Damit wäre auch schon das Ende des Blogs für diese Woche erreicht.
Wir freuen uns auch euer Feedback und werden euch nächste Woche von weiteren Fortschritte zu den Karten berichten.

**Comment by MelonigeMelone** [comment]
Guter und kurzer Blogeintrag. Ich bin bereits sehr gespannt wie die Implementierung der Karten aussieht, einerseits der Editor andererseits auch die technische Umsetzung. In den nächsten Blogeinträgen wäre es sehr interessant zu erfahren wie einzelne Funktionen auch technisch umgesetzt wurden, ich denke dabei vor allem an das Speichern und Lesen der Karten.
